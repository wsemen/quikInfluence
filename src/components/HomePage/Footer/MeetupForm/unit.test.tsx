import { render, screen } from '@testing-library/react';
import MeetUpForm from '.';

test('Should render form', async () => {
  const { container } = render(<MeetUpForm />);

  expect(container).toMatchSnapshot();
});
