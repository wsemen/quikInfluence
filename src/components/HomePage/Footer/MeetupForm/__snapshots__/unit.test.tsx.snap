// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render form 1`] = `
<div>
  <form
    class="css-1iq1sz"
  >
    <div
      class="css-12r6igv"
    >
      <div
        class="chakra-form-control css-15ucb8w"
        role="group"
      >
        <div
          class="css-79elbk"
        >
          <input
            aria-required="true"
            class="chakra-input css-mxr0mj"
            datatest-id="test-phoneNumber"
            id="field-1"
            name="phoneNumber"
            placeholder="Phone Number"
            required=""
            value=""
          />
          <svg
            aria-hidden="true"
            class="svg-inline--fa fa-phone fa-w-16 "
            color="red"
            data-icon="phone"
            data-prefix="fas"
            focusable="false"
            role="img"
            style="width: 10px; position: absolute; top: 50%; left: 30px; transform: translateY(-50%); z-index: 1;"
            viewBox="0 0 512 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M493.4 24.6l-104-24c-11.3-2.6-22.9 3.3-27.5 13.9l-48 112c-4.2 9.8-1.4 21.3 6.9 28l60.6 49.6c-36 76.7-98.9 140.5-177.2 177.2l-49.6-60.6c-6.8-8.3-18.2-11.1-28-6.9l-112 48C3.9 366.5-2 378.1.6 389.4l24 104C27.1 504.2 36.7 512 48 512c256.1 0 464-207.5 464-464 0-11.2-7.7-20.9-18.6-23.4z"
              fill="currentColor"
            />
          </svg>
        </div>
        
      </div>
      <div
        class="chakra-form-control css-15ucb8w"
        role="group"
      >
        <div
          class="css-79elbk"
        >
          <input
            class="chakra-input css-mxr0mj"
            datatest-id="test-isInfluencer"
            id="field-2"
            name="isInfluencer"
            placeholder="Are You an Influencer"
            value=""
          />
          <svg
            aria-hidden="true"
            class="svg-inline--fa fa-phone fa-w-16 "
            color="red"
            data-icon="phone"
            data-prefix="fas"
            focusable="false"
            role="img"
            style="width: 10px; position: absolute; top: 50%; left: 30px; transform: translateY(-50%); z-index: 1;"
            viewBox="0 0 512 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M493.4 24.6l-104-24c-11.3-2.6-22.9 3.3-27.5 13.9l-48 112c-4.2 9.8-1.4 21.3 6.9 28l60.6 49.6c-36 76.7-98.9 140.5-177.2 177.2l-49.6-60.6c-6.8-8.3-18.2-11.1-28-6.9l-112 48C3.9 366.5-2 378.1.6 389.4l24 104C27.1 504.2 36.7 512 48 512c256.1 0 464-207.5 464-464 0-11.2-7.7-20.9-18.6-23.4z"
              fill="currentColor"
            />
          </svg>
        </div>
        
      </div>
      <div
        class="chakra-form-control css-15ucb8w"
        role="group"
      >
        <div
          class="css-79elbk"
        >
          <input
            aria-required="true"
            class="chakra-input css-mxr0mj"
            datatest-id="test-name"
            id="field-3"
            name="name"
            placeholder="Your Name"
            required=""
            value=""
          />
          <svg
            aria-hidden="true"
            class="svg-inline--fa fa-user fa-w-14 "
            color="red"
            data-icon="user"
            data-prefix="fas"
            focusable="false"
            role="img"
            style="width: 10px; position: absolute; top: 50%; left: 30px; transform: translateY(-50%); z-index: 1;"
            viewBox="0 0 448 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M224 256c70.7 0 128-57.3 128-128S294.7 0 224 0 96 57.3 96 128s57.3 128 128 128zm89.6 32h-16.7c-22.2 10.2-46.9 16-72.9 16s-50.6-5.8-72.9-16h-16.7C60.2 288 0 348.2 0 422.4V464c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48v-41.6c0-74.2-60.2-134.4-134.4-134.4z"
              fill="currentColor"
            />
          </svg>
        </div>
        
      </div>
      <div
        class="chakra-form-control css-15ucb8w"
        role="group"
      >
        <div
          class="css-79elbk"
        >
          <input
            aria-required="true"
            class="chakra-input css-mxr0mj"
            datatest-id="test-company"
            id="field-4"
            name="company"
            placeholder="Company Name"
            required=""
            value=""
          />
          <svg
            aria-hidden="true"
            class="svg-inline--fa fa-user fa-w-14 "
            color="red"
            data-icon="user"
            data-prefix="fas"
            focusable="false"
            role="img"
            style="width: 10px; position: absolute; top: 50%; left: 30px; transform: translateY(-50%); z-index: 1;"
            viewBox="0 0 448 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M224 256c70.7 0 128-57.3 128-128S294.7 0 224 0 96 57.3 96 128s57.3 128 128 128zm89.6 32h-16.7c-22.2 10.2-46.9 16-72.9 16s-50.6-5.8-72.9-16h-16.7C60.2 288 0 348.2 0 422.4V464c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48v-41.6c0-74.2-60.2-134.4-134.4-134.4z"
              fill="currentColor"
            />
          </svg>
        </div>
        
      </div>
      <div
        class="chakra-form-control css-15ucb8w"
        role="group"
      >
        <div
          class="css-79elbk"
        >
          <input
            aria-required="true"
            class="chakra-input css-mxr0mj"
            datatest-id="test-email"
            id="field-5"
            name="email"
            placeholder="Your Email Address"
            required=""
            value=""
          />
          <svg
            aria-hidden="true"
            class="svg-inline--fa fa-mail-bulk fa-w-18 "
            color="red"
            data-icon="mail-bulk"
            data-prefix="fas"
            focusable="false"
            role="img"
            style="width: 10px; position: absolute; top: 50%; left: 30px; transform: translateY(-50%); z-index: 1;"
            viewBox="0 0 576 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M160 448c-25.6 0-51.2-22.4-64-32-64-44.8-83.2-60.8-96-70.4V480c0 17.67 14.33 32 32 32h256c17.67 0 32-14.33 32-32V345.6c-12.8 9.6-32 25.6-96 70.4-12.8 9.6-38.4 32-64 32zm128-192H32c-17.67 0-32 14.33-32 32v16c25.6 19.2 22.4 19.2 115.2 86.4 9.6 6.4 28.8 25.6 44.8 25.6s35.2-19.2 44.8-22.4c92.8-67.2 89.6-67.2 115.2-86.4V288c0-17.67-14.33-32-32-32zm256-96H224c-17.67 0-32 14.33-32 32v32h96c33.21 0 60.59 25.42 63.71 57.82l.29-.22V416h192c17.67 0 32-14.33 32-32V192c0-17.67-14.33-32-32-32zm-32 128h-64v-64h64v64zm-352-96c0-35.29 28.71-64 64-64h224V32c0-17.67-14.33-32-32-32H96C78.33 0 64 14.33 64 32v192h96v-32z"
              fill="currentColor"
            />
          </svg>
        </div>
        
      </div>
      <div
        class="chakra-form-control css-15ucb8w"
        role="group"
      >
        <div
          class="css-79elbk"
        >
          <input
            class="chakra-input css-mxr0mj"
            datatest-id="test-industry"
            id="field-6"
            name="industry"
            placeholder="Industry"
            value=""
          />
          <svg
            aria-hidden="true"
            class="svg-inline--fa fa-mail-bulk fa-w-18 "
            color="red"
            data-icon="mail-bulk"
            data-prefix="fas"
            focusable="false"
            role="img"
            style="width: 10px; position: absolute; top: 50%; left: 30px; transform: translateY(-50%); z-index: 1;"
            viewBox="0 0 576 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M160 448c-25.6 0-51.2-22.4-64-32-64-44.8-83.2-60.8-96-70.4V480c0 17.67 14.33 32 32 32h256c17.67 0 32-14.33 32-32V345.6c-12.8 9.6-32 25.6-96 70.4-12.8 9.6-38.4 32-64 32zm128-192H32c-17.67 0-32 14.33-32 32v16c25.6 19.2 22.4 19.2 115.2 86.4 9.6 6.4 28.8 25.6 44.8 25.6s35.2-19.2 44.8-22.4c92.8-67.2 89.6-67.2 115.2-86.4V288c0-17.67-14.33-32-32-32zm256-96H224c-17.67 0-32 14.33-32 32v32h96c33.21 0 60.59 25.42 63.71 57.82l.29-.22V416h192c17.67 0 32-14.33 32-32V192c0-17.67-14.33-32-32-32zm-32 128h-64v-64h64v64zm-352-96c0-35.29 28.71-64 64-64h224V32c0-17.67-14.33-32-32-32H96C78.33 0 64 14.33 64 32v192h96v-32z"
              fill="currentColor"
            />
          </svg>
        </div>
        
      </div>
    </div>
    <button
      class="chakra-button css-17mu9ok"
      type="button"
    >
      CONTACT US
    </button>
  </form>
</div>
`;
